WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyControlTileViewModel.cs

```diff
@@ -47,27 +42,35 @@ protected override void OnActivated(CompositeDisposable disposables)
 	{
 		base.OnActivated(disposables);
 
-		_walletVm.UiTriggers.PrivacyProgressUpdateTrigger
-			.Subscribe(_ => Update())
-			.DisposeWith(disposables);
+		_wallet.Privacy.Progress
+					   .CombineLatest(_wallet.Privacy.IsWalletPrivate)
+					   .CombineLatest(_wallet.Coins.List.ToCollection())
+					   .Flatten()
+					   .Throttle(TimeSpan.FromMilliseconds(250))
```

Comment:
I can see without the throttle it fires many many times... but I don't understand why. Can one of you explain? @ichthus1604 @SuperJMN

Reply:
I also see the tile is changing right after the wallet got logged in, probably caused by the throttle.
Can we avoid it somehow?

Reply:
@soosr the `Sample()` in `WalletModel.TransactionProcessed` was causing that. I've removed it in the latest commit. I don't think it makes any practical sense to keep it.

Reply:
It was added to `UiTriggers` here https://github.com/zkSNACKs/WalletWasabi/pull/10245, because the continous triggering caused UI freeze https://github.com/zkSNACKs/WalletWasabi/issues/10244.
If you remove it from WalletModel, it's fine because History still relies on UiTriggers.
Keep this in mind when you reach History with refactoring. https://github.com/zkSNACKs/WalletWasabi/pull/10245#pullrequestreview-1327766658